tests/Main.hs

```diff
@@ -45,16 +51,12 @@ mkInOutTests = do
       testCases <- for (Map.toList zippedPaths) $
         \(testName, InOut inFilePath outFilePath) -> do
           inFile <- readFile inFilePath
-          outFile <- readFile outFilePath
-          return $ testCase testName $
-            case showerString inFile of
-              Left parseError -> assertFailure parseError
-              Right s -> assertEqual "" (normalize s) (normalize outFile)
+          got <- case showerString inFile of
+            Left parseError -> assertFailure parseError
```

Comment:
Why did you move this failure out of the test case, to the test generation phase?

Reply:
Acidentaly.